Modules/FindPCOUVSDK.cmake

```diff
@@ -0,0 +1,79 @@
+###############################################################################
+# Find PCO UV SDK
+#
+#     find_package(PCOUVSDK)
+#
+# Variables defined by this module:
+#
+#  PCOUVSDK_FOUND                True if UV SDK found
+#  PCOUVSDK_VERSION              The version of UV SDK
+#  PCOUVSDK_INCLUDE_DIR          The location(s) of UV SDK headers
+#  PCOUVSDK_LIBRARY_DIR          Libraries needed to use UV SDK
+#  PCOUVSDK_BINARY_DIR           Binaries needed to use UV SDK
+
+SET(PCOUV_SDK_PATH_HINTS
+  "../ThirdParty/PCO_SDK"
```

Comment:
Update list of path hints, since SDK isn't in ThirdParty or Plus-bin.

Reply:
List of paths updated.